.bsp/sbt.json

```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.4.3","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/jt/Library/Application Support/JetBrains/IntelliJIdea2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
\ No newline at end of file
+{"name":"sbt","version":"1.4.3","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["C:\\Users\\david.gibbs\\.jdks\\corretto-1.8.0_275\\jre/bin/java","-Xms100m","-Xmx100m","-classpath","C:/Users/david.gibbs/AppData/Roaming/JetBrains/IdeaIC2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
I'm not entirely sure what this file is, I think it might be created by intellij, Could you remove this file?